### ISSUE TITLE
[a11y]: Detailed Grades

### ISSUE THREAD

**Issue by peterkulko** [openedx/frontend-app-learning#1883]
## Description

The weekly modules should be marked up as heading level 4s to support faster page navigation for people who use a screen reader.

PR: https://github.com/openedx/frontend-app-learning/pull/1813
